extensions/notebook/src/book/bookTrustManager.ts

```diff
@@ -27,7 +28,7 @@ export class BookTrustManager implements IBookTrustManager {
 		let trustableBookPaths = this.getTrustableBookPaths();
 		let hasTrustedBookPath: boolean = treeBookItems
 			.filter(bookItem => trustableBookPaths.some(trustableBookPath => trustableBookPath === path.join(bookItem.book.root, path.sep)))
-			.some(bookItem => normalizedNotebookUri.startsWith(path.join(bookItem.root, 'content', path.sep)));
+			.some(bookItem => normalizedNotebookUri.startsWith(bookItem.version === BookVersion.v1 ? path.join(bookItem.book.root, 'content', path.sep) : path.join(bookItem.book.root, path.sep)));
```

Comment:
So what happens when we get another book version? We have to go add another check to all these places to get the correct "root" path for the content? 

I know this isn't your change here but it would be great to start thinking about how to fix this in a better way - it seems like having the book item objects be instances of different book versions would make this whole thing a lot easier. So you could have v1Book, v2Book, etc that all extend the same base class but then override functionality like a function called `getContentRoot`.

@lucyzhang929 @chlafreniere FYI - something to think about